internal/artifacts/module_arch.go

```diff
@@ -64,6 +64,10 @@ func ExecutePack(source, target string, extensions []string, moduleName, platfor
 		return nil
 	}
 
+	if module.Path == "" {
```

Comment:
This should also be handled during the verbose build

Reply:
added